system-variables.md

```diff
@@ -1758,20 +1758,6 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 默认值：`OFF`
 - 该变量指定 TiDB 是否收集扩展统计信息来指导优化器。
 
-### `tidb_enable_fast_analyze`
```

Comment:
Does the item `tidb_enable_fast_analyze` in the output of `SHOW GLOBAL VARIABLES LIKE 'tidb%';` need to be deleted?

https://github.com/pingcap/docs-cn/blob/2c92925651c1b7e137fe4f75ee8beec8026dcbca/sql-statements/sql-statement-show-variables.md?plain=1#L65

Reply:
Updated.